.github/workflows/codecov-ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Code Coverage
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+concurrency:
+  group: code-cov-${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+jobs:
+  codecov-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install lcov
+        run: sudo apt-get -y install lcov
+      - uses: actions/checkout@master
```

Comment:
what's `master` here?

Reply:
I believe it's referring to a branch on the [actions/checkout repo](https://github.com/actions/checkout/tree/master). Good catch, I'll change it to [`v4`](https://github.com/actions/checkout/releases/tag/v4).